Groups: Update groups layout

KosFurler opened the pull request.

veselcraft committed:
Groups: Update groups layot

veselcraft committed:
We don't use crowdin anymore, why it's still there?

veselcraft reviewed (changes requested):
Проблемы с табами. Огромные. Советую погуглить `Beautifier` в гугле для VS Code

Алсо, было бы неплохо перевести новые приколы и на английский язык

Commit:
Locales: Delete unnecessary LICENSE

We already have COPYING in the root directory :/

Commit:
Locales: Localize strings for updated group layout

Localized strings: open_new_group, open_group_desc, search_group, search_by_groups, search_group_desc
(we already have search_for_groups but idk, I'll leave it for now)

Commit:
Global: Fix tabs

Commit:
Groups: Fix tabs and div block

I have SO many questions...

maksalees reviewed (approved):
dsrev пофиксил все, так что апрув

veselcraft reviewed (approved):
Да, всё оки

KosFurler committed:
Groups: Update layot

New counter, info and actions blocks and some stye fixes

Commit:
Global: Fix tabs

Commit:
Locales: Localize strings for updated group layout

Localized strings: check_community, groups_list

Commit:
Merge branch 'master' into similartovk

Commit:
Update Groups.xml

celestora committed:
Update Groups.xml
